manifests/kots-helm.yaml

```diff
@@ -561,3 +569,15 @@ spec:
         tenantRequirementsChecker: *customSecurityContext
         internalPostgresql: *customSecurityContext
         upgradeCheck: *customSecurityContext
+
+    # Airgap mode AND Local Registry
+    - when: '{{repl and (IsAirgap) (HasLocalRegistry) }}'
+      recursiveMerge: true
+      values:
+        global:
+          imageRegistry: '{{repl LocalRegistryAddress }}'
+          imagePullSecrets:
+            - '{{repl ImagePullSecretName }}'
+        replicated:
+          imagePullSecrets:
+            - '{{repl ImagePullSecretName }}'
```

Comment:
We shared some feedback with Replicated team as IMHO the replicated helm chart should be able to reuse the global image pull secrets https://cartoteam.slack.com/archives/C05V5V84W7M/p1718285356392639?thread_ts=1718211262.868209&cid=C05V5V84W7M